qutebrowser/test/mainwindow/conftest.py

```diff
@@ -0,0 +1,34 @@
+"""
+pytest fixtures and utilities for testing.
+
+Fixtures defined here will be visible to all test files in this directory and
```

Comment:
Wouldn't it make more sense to have this `default_config` fixture in the main `conftest.py` rather than in `mainwindow/conftest.py`? Using the config is something which isn't really specific to the main window :wink: